app/views/products/_form_git.html.erb

```diff
@@ -1,5 +1,5 @@
 <div class="field">
-  <%= f.text_field :git_remote, help: "Example: <strong>git@github.com:Example/example.git</strong> (<a href='https://help.github.com/articles/generating-an-ssh-key/' target='_blank'>requires SSH key installed</a>)".html_safe %>
+  <%= f.text_field :git_remote, help: "Example: <strong>git@github.com:Example/example.git</strong> (<a href='https://help.github.com/articles/generating-an-ssh-key/' target='_blank'>requires SSH key installed</a>) or <strong>https://github.com/Example/</strong>".html_safe %>
```

Comment:
If the example for ssh is `git@github.com:Example/example.git` then the example for https would likely be `https://github.com/Example/example.git` rather than `https://github.com/Example/`.

i.e.  git@github.com:OSC/ood-dashboard.git and https://github.com/OSC/ood-dashboard.git